api/goldens/v1/aptos_api__tests__blocks_test__test_get_genesis_block_by_version.json

```diff
@@ -186,7 +186,7 @@
           "data": {
             "type": "0x1::coin::CoinInfo<0x1::aptos_coin::AptosCoin>",
             "data": {
-              "decimals": "8",
+              "decimals": 8,
```

Comment:
Did you change this manually or did the json serialization get better?

Reply:
If it's a u64 it's converted to a string. u8 is fine being a number

Reply:
haha if it's < uint64, it serializes as an actual int. JS max is 2**53-1, so no chance of it getting better any time soon ;-)